ray-operator/config/samples/ray-cluster.complete.yaml

```diff
@@ -97,8 +106,16 @@ spec:
             name: ray-code
             # An array of keys from the ConfigMap to create as files
             items:
-            - key: code.py
-              path: code.py
+            - key: example.py
+              path: example.py
+        - name: autoscaler-config
```

Comment:
I think you are playing with configuration here to save some debugging efforts on e2e work? 
Just like to confirm if we are on the same page on the flow. 

if `EnableInTreeAutoscaling` is enabled, 

1. Controller converts CR to autoscaling config. 
2. Controller create configmap and mount to head automatically

Reply:
Yes, this will be integrated into the controller, it is just for testing at the moment.